README.md

```diff
@@ -19,15 +19,27 @@ Ensure you have the following installed on your machine:
 -   [Docker](https://www.docker.com/get-started)
 -   [Docker Compose](https://docs.docker.com/compose/install/)
 
-### Run
+### Setup
 
-To deploy locally with Docker, run the following command:
+1. Build and start the services in detached mode:
 
 ```
 docker-compose up -d --build
 ```
 
-Once the deployment is complete, you can access Hasura by navigating to:
+2. After starting Docker Compose, run the following command to apply the database migrations:
```

Comment:
i think we should clarify that you need to setup the `.env` file inside `apps/scripts` folder before running the scripts